src/Microsoft.VisualStudio.ProjectSystem.Managed/GlobalSuppressions.cs

```diff
@@ -1,3 +1,4 @@
 ﻿[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "VSTHRD110:Observe result of async calls", Justification = "https://github.com/dotnet/project-system/issues/3921", Scope = "member", Target = "~M:Microsoft.VisualStudio.ProjectSystem.JoinableTaskQueue.Register(Microsoft.VisualStudio.Threading.JoinableTask)")]
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "VSTHRD004:Await SwitchToMainThreadAsync", Justification = "https://github.com/Microsoft/vs-threading/issues/377", Scope = "member", Target = "~M:Microsoft.VisualStudio.ProjectSystem.ThreadingExtensions.SwitchToUIThread(Microsoft.VisualStudio.ProjectSystem.IProjectThreadingService,System.Threading.CancellationToken)~Microsoft.VisualStudio.Threading.JoinableTaskFactory.MainThreadAwaitable")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1068:CancellationToken parameters must come last", Justification = "https://github.com/dotnet/roslyn-analyzers/issues/1816", Scope = "member", Target = "~M:Microsoft.VisualStudio.ProjectSystem.OnceInitializedOnceDisposedUnderLockAsync.TryTranslateException(System.Exception,System.Threading.CancellationToken,System.Threading.CancellationToken)~System.Boolean")]
```

Comment:
The bug that requires suppression this has been fixed in https://github.com/dotnet/roslyn-analyzers/pull/1913.